cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateMaskedPaths(spec *rspec.Spec) error {
+	fmt.Println("validating maskedPaths")
+	for _, v := range spec.Linux.MaskedPaths {
+		fi, err := os.Stat(v)
+		if err != nil {
+			return err
+		}
+		if fi.Mode()&0444 != 0 {
+			return fmt.Errorf("%v should be readonly", v)
```

Comment:
Ah, maybe we need "just try it". I get from runc code, it implement maskedpaths as bing mount them to /dev/null